types/validator/index.d.ts

```diff
@@ -13,1237 +13,1237 @@
-    | 'sk-SK'
-    | 'sr-RS@latin'
-    | 'sr-RS'
-    | 'sv-SE'
-    | 'tr-TR'
-    | 'uk-UA'
-    | 'ku-IQ'
-    | 'ar'
-    | 'he'
-    | 'fa-IR'
-    | 'en-AU'
-    | 'en-GB'
-    | 'en-HK'
-    | 'en-IN'
-    | 'en-NZ'
-    | 'en-ZA'
-    | 'en-ZM'
-    | 'ar-AE'
-    | 'ar-BH'
-    | 'ar-DZ'
-    | 'ar-EG'
-    | 'ar-IQ'
-    | 'ar-JO'
-    | 'ar-KW'
-    | 'ar-LB'
-    | 'ar-LY'
-    | 'ar-MA'
-    | 'ar-QM'
-    | 'ar-QA'
-    | 'ar-SA'
-    | 'ar-SD'
-    | 'ar-SY'
-    | 'ar-TN'
-    | 'ar-YE'
-    | 'pt-BR'
-    | 'pl-Pl';
-
-export const isAlphaLocales: AlphaLocale[];
-
-/**
- * Check if the string contains only letters (a-zA-Z).
- *
- * @param [locale] - AlphaLocale
- */
-export function isAlpha(str: string, locale?: AlphaLocale): boolean;
-
-export type AlphanumericLocale =
-    | 'en-US'
-    | 'bg-BG'
-    | 'cs-CZ'
-    | 'da-DK'
-    | 'de-DE'
-    | 'el-GR'
-    | 'es-ES'
-    | 'fr-FR'
-    | 'it-IT'
-    | 'hu-HU'
-    | 'nb-NO'
-    | 'nl-NL'
-    | 'nn-NO'
-    | 'pl-PL'
-    | 'pt-PT'
-    | 'ru-RU'
-    | 'sl-SI'
-    | 'sk-SK'
-    | 'sr-RS@latin'
-    | 'sr-RS'
-    | 'sv-SE'
-    | 'tr-TR'
-    | 'uk-UA'
-    | 'ku-IQ'
-    | 'ar'
-    | 'he'
-    | 'fa-IR'
-    | 'en-AU'
-    | 'en-GB'
-    | 'en-HK'
-    | 'en-IN'
-    | 'en-NZ'
-    | 'en-ZA'
-    | 'en-ZM'
-    | 'ar-AE'
-    | 'ar-BH'
-    | 'ar-DZ'
-    | 'ar-EG'
-    | 'ar-IQ'
-    | 'ar-JO'
-    | 'ar-KW'
-    | 'ar-LB'
-    | 'ar-LY'
-    | 'ar-MA'
-    | 'ar-QM'
-    | 'ar-QA'
-    | 'ar-SA'
-    | 'ar-SD'
-    | 'ar-SY'
-    | 'ar-TN'
-    | 'ar-YE'
-    | 'pt-BR'
-    | 'pl-Pl';
-
-export const isAlphanumericLocales: AlphanumericLocale[];
-
-/**
- * Check if the string contains only letters and numbers.
- *
- * @param [locale] - AlphanumericLocale
- */
-export function isAlphanumeric(str: string, locale?: AlphanumericLocale): boolean;
-
-/**
- * Check if the string contains ASCII chars only.
- */
-export function isAscii(str: string): boolean;
-
-/**
- * Check if a string is base32 encoded.
- */
-export function isBase32(str: string): boolean;
-
-/**
- * Check if a string is base64 encoded.
- */
-export function isBase64(str: string): boolean;
-
-/**
- * Check if the string is a date that's before the specified date.
- *
- * @param [date] - Date string (defaults to now)
- */
-export function isBefore(str: string, date?: string): boolean;
-
-/**
- * Check if a string is a BIC (Bank Identification Code) or SWIFT code.
- */
-export function isBIC(str: string): boolean;
-
-/**
- * check if a string is a boolean.
- */
-export function isBoolean(str: string): boolean;
-
-export interface IsByteLengthOptions {
-    /**
-     * @default 0
-     */
-    min?: number;
-    /**
-     * @default undefined
-     */
-    max?: number;
-}
-
-/**
- * Check if the string's length (in UTF-8 bytes) falls in a range.
- *
- * @param [options] - Options
- */
-export function isByteLength(str: string, options?: IsByteLengthOptions): boolean;
+declare namespace validator {
```

Comment:
I just moved everything in this file inside a namespace and changed the export statement at the bottom